src/core/thread/mlr_manager.cpp

```diff
@@ -67,10 +68,36 @@ MlrManager::MlrManager(Instance &aInstance)
 void MlrManager::HandleNotifierEvents(Events aEvents)
 {
 #if OPENTHREAD_CONFIG_MLR_ENABLE
+    uint8_t                aSequenceNumber = 0;
```

Comment:
```suggestion
    uint8_t                sequenceNumber = 0;
```
The `a` prefix is reserved for arguments.